libraries/stumbler/build.gradle

```diff
@@ -145,6 +145,13 @@ dependencies {
         exclude group: 'org.json'
     }
 
+    // marisa-trie for offline geo support
+    compile 'org.mozilla.mozstumbler.marisa:marisa:0.4'
+
+    // Google Guava.  I'm lazy and want to use collections that aren't
+    // crazy.
+    compile group: 'com.google.guava', name: 'guava', version: '12.0'
```

Comment:
The guava dependency is no longer required as we no longer use set intersections or any collections whatsoever for private location fixes.